javascript/modules/client/test/integration/client.test.ts

```diff
@@ -131,46 +136,53 @@ describe("Client", () => {
   });
 
   describe("DAO Creation", () => {
-    test.todo("Should estimate gas fees for creating a DAO");
-    // it("Should estimate gas fees for creating a DAO", async () => {
-    //   const context = new Context(contextParamsLocalChain);
-    //   const client = new Client(context);
-
-    //   const daoCreationParams: ICreateParams = {
-    //     daoConfig: {
-    //       name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
-    //       metadata: "0x1234",
-    //     },
-    //     plugins: [
-    //       { id: "0x1234", data: "0x1234" },
-    //     ],
-    //     gsnForwarder: Wallet.createRandom().address,
-    //   };
-
-    //   const gasFeesEstimation = await client.estimation.create(
-    //     daoCreationParams,
-    //   );
-
-    //   expect(typeof gasFeesEstimation).toEqual("object");
-    //   expect(typeof gasFeesEstimation.average).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toBeGreaterThan(BigInt(0));
-    //   expect(gasFeesEstimation.max).toBeGreaterThan(gasFeesEstimation.average);
-    // });
+    it("Should estimate gas fees for creating a DAO", async () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const daoName = "ERC20VotingDAO_" + Math.floor(Random.getFloat() * 9999) + 1
+
+      const daoCreationParams: ICreateParams = {
+        metadata: {
+          name: daoName,
+          description: "this is a dao",
+          avatar: 'https://...',
+          links: []
+        },
+        ensSubdomain: daoName,
```

Comment:
This should always be a lowercase, ASCII string with dashes, but not any random thing